product_docs/docs/epas/16/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/05_dbms_job/index.mdx

```diff
@@ -11,6 +11,8 @@ redirects:
 
 The `DBMS_JOB` package lets you create, schedule, and manage jobs. A job runs a stored procedure that was previously stored in the database. The `SUBMIT` procedure creates and stores a job definition. A job identifier is assigned to a job with a stored procedure and the attributes describing when and how often to run the job.
 
+The `edb_job_scheduler` is an extension that runs a job scheduler as a background process for the `DBMS_JOB` package. For more information, see [EDB Job Scheduler extension](/pg_extensions/edb_job_scheduler).
```

Comment:
Possible re-order for clarity...

The `DBMS_JOB` package's jobs are scheduled and run in the background by the `edb_job_scheduler` extension.